simulations/utils.py

```diff
@@ -43,51 +45,26 @@ def select_protocol(protocol):
         return IntegerProtocol
 
 
-def message_maker(mode):
-    """The message maker defines the logic for running each type of simulation."""
+def message_strategy(mode):
```

Comment:
Does it make sense to keep these a closures, rather than objects? I made the closures initially, but I'm not sure this will be totally appropriate in the future (as we want what this returns to be parameterized based on a view...)

For example, I think we might want to have a BasicStrategy class, and "Strategies" that inherit from this. They take a view (plus whatever else they need - maybe a clock?), and then you can call the function "should I make a message right now."

Reply:
Moving toward objects. Just did this for a quick integration